scss/titles.scss

```diff
@@ -0,0 +1,23 @@
+/*    title-decoration-height: 90px;
```

Comment:
remove commented code and btw if you want to add comments use `//` because those are scss comments and are removed when compiling to css while `/* */` comments will be present in the final css.